feature/sdk-nativa (#2612)
mercadopago/devsite-docs: guides/snippets/api-mobile/ios.en.md

    )
}
```

The parameters are listed in the table below.

| Parameter         | Type    | Description                                                                                                | Required | 
|-------------------|---------|------------------------------------------------------------------------------------------------------------|----------| 
| bin               | String  | The first 8 digits of the credit card, obtained by the `onBinChange` callback from `CardNumberTextField`.  | Required | 
| paymentMethodId   | String  | Payment method ID, obtained in the result of the `PaymentMethods` method.                                 | Required | 

For more information about the response, check the [method documentation on GitHub](https://mercadopago.github.io/sdk-ios/latest/documentation/coremethods/coremethods/issuers(bin:paymentmethodid:)).

:::

:::AccordionComponent{title="Get document types" pill="client-side"} 
Mercado Pago requires the validation of a cardholder's document. This method returns all accepted document types, such as CPF, RG, DNI, for the selected country.

```swift
Task {
    let documents = await 
self.coreMethods.identificationTypes()
}
```

> Para mais informações sobre a resposta da chamada, consulte a [documentação do método no GitHub](https://mercadopago.github.io/sdk-ios/latest/documentation/coremethods/coremethods/identificationtypes()).

Reviewer: ```suggestion
For more information about the call response, check the [method documentation on GitHub](https://mercadopago.github.io/sdk-ios/latest/documentation/coremethods/coremethods/identificationtypes/).
```
